### PR TITLE
Update about text

### PR DESCRIPTION
**Pull request by NyanHelsing**
This updates the about text on the dashboard to the latest version provided by UCSD.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+JSONStream@^1.0.3:
```
Comment (samanehsan) [inline yarn.lock]: Was this `yarn.lock` file mistakenly added into this PR?

**Comment by samanehsan** [comment]
@birdbrained, I have one question about the `yarn.lock` file but besides that the rest of it looks good :octocat:

File: yarn.lock (hunk 1 above)
Comment (birdbrained) [inline yarn.lock]: yarn.lock should be committed.

Comment (samanehsan) [inline yarn.lock]: Ok, yarn is for dependency management right? I'm not sure I understand why is it necessary to include in this PR that just changes text.

**Pull request merged**